test/unit/puppet_fact_parser_test.rb

```diff
@@ -473,6 +485,14 @@ def structured_networking_facts
     read_json_fixture('facts/facts_structured_networking.json')['facts']
   end
 
+  def example_v3_facts
```

Comment:
Just want to point out that [facterdb](https://github.com/camptocamp/facterdb) exists. Not asking for a change, but it may be good to know it's out there. Sadly, it looks like no Facter 4 facts are present right now.